Tests/SourceKittenFrameworkTests/ModuleTests.swift

```diff
@@ -10,7 +10,7 @@ let projectRoot: String = bazelProjectRoot ?? #file.bridge()
 
 class ModuleTests: XCTestCase {
 
-#if !os(Linux)
+#if !(os(Linux) || os(Windows))
```

Comment:
Since Xcode is only available on macOS, can you please change this to `#if os(macOS)`? I probably should have done that initially too.

Reply:
Done with #780.